website/templates/dark_mode_toggle.html

```
@@ -0,0 +1,6 @@
+<button id="theme-toggle"
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)